packages/fluentui/react-northstar/src/themes/teams/components/Breadcrumb/breadcrumbItemStyles.ts

```diff
@@ -0,0 +1,12 @@
+import { BreadcrumbItemStylesProps } from '../../../../components/Breadcrumb/BreadcrumbItem';
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+
+export const breadcrumbItemStyles: ComponentSlotStylesPrepared<BreadcrumbItemStylesProps, BreadcrumbVariables> = {
+  root: ({ props: p, variables: v, theme: { siteVariables } }): ICSSInJSStyle => {
+    return {
+      display: 'inline-block',
+    };
+  },
+
+  breadcrumb: ({ variables: v }) => ({ zIndex: v.breadcrumbZIndex }),
```

Comment:
```suggestion
```

I don't think that there is `breadcrumb` slot in `Breadcrumb` item

Reply:
fixed by commit [feat(Breadcrumb): extract style variable](https://github.com/microsoft/fluentui/pull/14926/commits/e1bbef872ed494f842153450a4bff21ca5878f0b)